Resources/Prototypes/Roles/Jobs/Cargo/quartermaster.yml

```diff
@@ -1,13 +1,13 @@
 - type: job
   id: Quartermaster
   name: "quartermaster"
-  head: true 
+  weight: 10
```

Comment:
merge mistake?

Reply:
or was `head` removed, I can't tell? what did it even do?

Reply:
`head` is no longer a field. All it did was give a job higher priority, and `weight` replaces that functionality.